gettsim/config.py

```diff
@@ -127,6 +124,10 @@
     "m_kind_berücks_zeit": float,
     "m_pfleg_berücks_zeit": float,
     "y_pflichtbeitr_ab_40": float,
+    "anwartschaftszeit": bool,
+    "arbeitssuchend": bool,
+    "arbeitsl_geld_bezug_m": int,
+    "versich_pfl_m": int,
```

Comment:
```suggestion
    "soz_vers_pflicht_5j": int,
```
I first thought "Versicherung Pflege" without having seen any context :smile: 

We are already using `soz_vers_` elsewhere, I am not married to the rest of the name except there should be no `_m` ending (reserved for measurements at the monthly level as opposed to the unit being a month)

Reply:
sounds good 👍